docs/source/platform/federation.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Federation in the Apollo Platform
+description: Use all of the features of the Apollo Platform with a federated GraphQL architecture
+---
+[//]: # (Description: Introductory content, discussing the overall model of the federation platform, why you would want to register partial schemas, and framing the concept of thinking about running federation as a coordination and collaboration problem to motivate the tools we've built)
+[//]: # (Assignee: Chang)
+[//]: # (Reviewer: Caydie, James)
+
+The key difference between running GraphQL with a monolithic server instead of running multiple services orchestrated by a [gateway](https://www.apollographql.com/docs/apollo-server/federation/implementing/#running-a-gateway) is that your GraphQL schema is dynamically defined by composing an overall schema from the [partial schemas](<LINK_TO_GLOSSARY>) of GraphQL services that comply with the [federation spec](https://www.apollographql.com/docs/apollo-server/federation/federation-spec/). This empowers teams to own a portion of the graph, and move faster by allowing changes to be independently developed and deployed.  
+
+The Apollo GraphQL platform provides tools to help ensure that this increase in autonomy doesn't come at a cost to stability. Since the GraphQL schema is the central hub of the data in your graph, it's important to [validate](<LINK_TO_SCHEMA_VALIDATION>) that any changes to a particular service's partial schema won't cause breakages in other parts of your graph, or affect existing [operations made by clients](LINK_TO_METRICS), so you can scale your data graph with confidence.
+
+## Inspecting your Graph
+> Using the `apollo service:list` command or the UI to understand your graph
+
+To understand the implementing services that make up your graph, you can use both the Apollo command line interface as well as the Apollo Platform.
+
+Use `apollo service:list` in the command line to see a snapshot of the services that make up your graph, including their endpoints and when they were last updated.
+
+Here's what running `apollo service:list` will look like:
+
+```
+~$ apollo service:list
+  ✔ Loading Apollo Project
+  ✔ Fetching list of services for graph service-list-federation-demo
+
+name       URL                            last updated
+─────────  ─────────────────────────────  ────────────────────────
+Accounts   http://localhost:4001/graphql  3 July 2019 (2 days ago)
+Inventory  http://localhost:4004/graphql  3 July 2019 (2 days ago)
```

Comment:
Do we order the service lists in any way? Like are they ordered by last updated? If they are it might cool to have different last updated entries so readers can see which ones were updated recently

Reply:
They are alphabetical in the UI, which we can do here. I think we should keep it consistent between UI and CLI, no?